### PR TITLE
Add a script to regenerate all example model files for test

### PR DESCRIPTION
**Pull request by hwpang**
## Description
Sometimes we need to regenerate example model files when we make module level changes. It can be annoying to do so manually every time. I added a script to update all example model files used in tests automatically.

### CODE REVIEW

File: tests/regenerate_models.sh [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/bin/bash -l
+
+mamba activate <chemprop_env>
```
Comment (am2145) [inline tests/regenerate_models.sh:3]: I think we only specify 'conda' when installing chemprop currently.

**Review by am2145 — COMMENTED** [review]
Aside from the mamba vs conda, everything LGTM.

**Comment by davidegraff** [comment]
this script relies on users modifying it for it to work properly, but we likely don't want them committing these changes. It's a weird catch-22 where we're now tracking the file, but also _not tracking_ any changes to it? We should think of a more programmatic way to achieve this. One quick fix would be to accept these two variables as input arguments:

```sh
CHEMPROP_ENV=$1
PATH_TO_CHEMPROP=$2

if [ -z "${CHEMPROP_ENV}"] || [ -z "${PATH_TO_CHEMPROP}" ]; then
    exit 1
```

**Review by am2145 — APPROVED** [review]
I see you have made mine and @davidegraff's suggested changes so I will approve this and set it to auto-merge.

**Pull request merged**